libraries/cmake/source/librdkafka/config/x86_64/linux/config.h

```diff
@@ -7,12 +7,19 @@
 #define CPU "generic"
 #define WITHOUT_OPTIMIZATION 0
 #define ENABLE_SSL 1
-#define ENABLE_GSSAPI 0
+#define ENABLE_GSSAPI 1
 #define ENABLE_DEVEL 0
 #define ENABLE_VALGRIND 0
 #define ENABLE_REFCNT_DEBUG 0
 #define ENABLE_SHAREDPTR_DEBUG 0
 #define ENABLE_LZ4_EXT 0
+#define ENABLE_LZ4_EXT 0
+#define ENABLE_SSL 1
+#define ENABLE_GSSAPI 1
+#define ENABLE_GSSAPI 1
+#define ENABLE_LZ4_EXT 0
+#define ENABLE_LZ4_EXT 0
+#define WITH_STATIC_LINKING 1
```

Comment:
`#define ENABLE_GSSAPI 1` , `#define ENABLE_SSL 1`, `#define ENABLE_LZ4_EXT 0` are defined multiple times successively, is this intentional?